lib/spoom/cli/commands/bump.rb

```diff
@@ -0,0 +1,61 @@
+# typed: strict
+# frozen_string_literal: true
+
+require 'find'
+require 'open3'
+
+require_relative 'base'
+
+module Spoom
+  module Cli
+    module Commands
+      class Bump < Base
+        extend T::Sig
+
+        default_task :bump
+
+        desc "bump", "change Sorbet sigils from one strictness to another when no errors"
+        option :from, type: :string
+        option :to, type: :string
+        option :force, type: :boolean, default: false, aliases: :f
```

Comment:
We can use `:desc` to provide a description of what this option does.

Reply:
Does that go above each option? Or only for `:force`?

Reply:
at least `:force`